docs/src/contrib/build.md

```diff
@@ -15,6 +15,20 @@ git clone git@github.com:metacontroller/metacontroller.git metacontroller
 cd metacontroller
 ```
 
+Then you can build a `metacontroller` binary like so:
+
+```sh
+make build
+```
+
+Note that you will need the following k8s code-generation tools:
```

Comment:
it will be automatically downloaded by Makefile - https://github.com/metacontroller/metacontroller/blob/master/Makefile#L73

Reply:
it wasn't in your case ?

Reply:
ah it was put in my $GOPATH which is not in my path (i don't normally install things this way, preferring to only install things using my OS package manager). i'll remove this as i will be in the minority here for sure